themes/theme-ios11/locale/it-IT.json

```diff
@@ -244,19 +244,19 @@
   },
   "navigation": {
     "login_register": "Accedi / Registrati",
-    "your_account": "Il tuo Account",
+    "your_account": "Il mio account",
```

Comment:
hier ist auch Durcheinander in den Sprachen zwischen "mein Account" und "Dein Account".. vielleicht PO fragen?